src/DbConnection.php

```diff
@@ -0,0 +1,25 @@
+<?php
+namespace Pokedex;
+use \PDO;
+
+
+class DbConnection
+{
+    private $db;
+    /**
+     * DbConnection constructor creates new connection to the pokedex_hb database
+     */
+    public function __construct()
+    {
+        $this->db =  new PDO('mysql:dbname=pokedex_hb;host=127.0.0.1','root');
```

Comment:
its common practise although not required, to put the DB name, host and username into constants in this class